spec/models/course/notification_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Course::Notification, type: :model do
```

Comment:
This should be in notification_spec. But when I test `notify` in activity, this could also be tested. So, do I need to test it again in notification_spec?